packages/discord.js/src/client/websocket/WebSocketManager.js

```diff
@@ -242,7 +242,7 @@ class WebSocketManager extends EventEmitter {
     this._ws.on(WSWebSocketShardEvents.Closed, ({ code, shardId }) => {
       const shard = this.shards.get(shardId);
       shard.emit(WebSocketShardEvents.Close, { code, reason: reasonIsDeprecated, wasClean: true });
-      if (UNRESUMABLE_CLOSE_CODES.includes(code) && this.destroyed) {
+      if (UNRESUMABLE_CLOSE_CODES.includes(code)) {
```

Comment:
The array also includes `1000` - a normal close that _will_ try to reconnect unless the ws is destroyed. We'll need to handle that manually